server_test.go

```diff
@@ -37,6 +37,11 @@ type AllowHook struct {
 	HookBase
 }
 
+func (h *AllowHook) SetOpts(l *slog.Logger, opts *HookOptions) {
```

Comment:
I haven't figured out where the AllowHook.SetOpts() and AllowHook.SetOpts() functions are being used.

Reply:
I don't think they currently are in the hooks. They are probably there as placeholder.